.travis.yml

```diff
@@ -6,6 +6,8 @@ go:
     - 1.4
     - 1.5
     - 1.6
+    - 1.7
+    - latest
```

Comment:
Can you point out where this feature is documented? I am not able to find it on https://docs.travis-ci.com/user/languages/go page, I only see the following note there:

> ### Specifying a Go version to use
>
> You can use any tagged version of Go or use `tip` to get the latest version.

Is `latest` documented somewhere else?